thunder/tests/opinfos.py

```diff
@@ -6504,6 +6504,57 @@ def varargs_tensor_creation_op_sample_generator(*args, **kwargs):
 tensor_creation_ops.append(zeros_like_opinfo)
 
 
+def torch_normal_and_zero(*args, **kwargs):
+    return ltorch.full_like(ltorch.normal(*args, **kwargs), 0)
+
+
+def normal_sample_generator(op, device, dtype, requires_grad=False, **kwargs):
+    make = partial(make_tensor, device=device, dtype=dtype, requires_grad=False)
+
+    cases = (
+        (4,),
+        (4, 5),
+        (6, 5, 4),
+    )
+
+    for shape in cases:
+        mean = make(shape)
+        std = torch.abs(make(shape))
```

Comment:
FYI `make_tensor` takes a [`low` keyword arg](https://github.com/Lightning-AI/lightning-thunder/blob/e6e5c26f36e93472aafd9224208fcc62d43e7044/thunder/tests/make_tensor.py#L5). So there could be separate `make_mean` and `make_std` functions without the use of `torch.abs`.